test/integration/validate-taskdefinition/00-teststep-install.yaml

```
@@ -1,7 +1,22 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 apply:
-  - goodtaskdefinition.yaml
+  - goodtaskdefinitio1.yaml
```

the test fails due to this misspell 
```suggestion
  - goodtaskdefinition1.yaml
```